### DISCUSSION TITLE
Not working

### DISCUSSION THREAD

**Discussion by myrnavero** [AveYo/ShowMMR#23] [General]
Hi!
Its not working after new compendium update. It worked before this patch.
Reinstalled showmmr according to the instructions, but it didnt help

**Comment by AveYo** [comment]
Finally updated the mod, should work now.
Probably need to run the standalone ShowMMR Tool to refresh the missing games

**Reply by myrnavero** [reply]
ok. thank u very much, you have a very useful mod

**Reply by myrnavero** [reply]
> Finally updated the mod, should work now. Probably need to run the standalone ShowMMR Tool to refresh the missing games

Hi
Its not workin again.

**Comment by ExivMyLife** [comment]
No matter how I try, mmr is not displayed after September (my friends have the same thing, and the 10k player who told me about it). Old games are displayed, new ones are not.
![1](https://github.com/AveYo/ShowMMR/assets/149298897/9c90e2db-255c-46f5-a2dd-4a0458f8d531)

**Reply by ExivMyLife** [reply]
Now instead of user_keys_%d_slot3.vcfg the file is user_keys_0_slot3.vcfg was this the way it was or is it now the way it is, is this the problem))?  (well, with my ID there is also a file, but the file with keys_0 is updated)?
